charts/app-config/values-integration.yaml

```
@@ -1109,10 +1109,13 @@ govukApplications:
         <<: *alb-ingress-www-waf-ruleset
         alb.ingress.kubernetes.io/load-balancer-name: assets-origin
         alb.ingress.kubernetes.io/group.name: assets-origin
-        alb.ingress.kubernetes.io/group.order: '16'
+        alb.ingress.kubernetes.io/group.order: '15'
         alb.ingress.kubernetes.io/security-groups: eks_ingress_www_origin
         alb.ingress.kubernetes.io/load-balancer-attributes: *assets-lb-attributes
-        alb.ingress.kubernetes.io/conditions.{{ .Release.Name }}: *assets-lb-conditions
+        alb.ingress.kubernetes.io/conditions.{{ .Release.Name }}: &draft-assets-lb-conditions >
```

Remove unused anchor
```suggestion
        alb.ingress.kubernetes.io/conditions.{{ .Release.Name }}: >
```

---

Thanks.  Will remove this.